UefiTestingPkg/Include/Library/FlatPageTableLib.h

```diff
@@ -0,0 +1,166 @@
+/** @file
+  Library to parse page/translation table entries.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PAGE_TABLE_ATTRIBUTE_LIB_H_
+#define PAGE_TABLE_ATTRIBUTE_LIB_H_
+

```

Comment:
do these structures need #pramga packed?

Reply:
It seems like they should be but because these structures mirror those in CpuPageTableLib (which doesn't #pragma pack) I opted not to

Reply:
Just to complete this conversation because I think the answer could be better/more educational for the team.  Packing is one of those odd things is often overdone.  @apop5  - why do they need packed?  

I don't think they need packed because you are not using them across compiler boundaries, and they are not being serialized or transported to other environments?  

It is important to note that there will be 32bits of wasted space added to the page_map to make up for the 32bit/64bit issue.  I doubt the entry will have anything added.